libutils/known_dirs.c

```diff
@@ -106,6 +105,8 @@ GET_DEFAULT_DIRECTORY_DEFINE(Master, master, MASTERDIR, "masterfiles")
 GET_DEFAULT_DIRECTORY_DEFINE(Input, input, INPUTDIR, "inputs")
 GET_DEFAULT_DIRECTORY_DEFINE(State, state, STATEDIR, "state")
 
+#endif
```

Comment:
This seems to be breaking mingw builds? https://ci.cfengine.com/job/testing-pr/label=PACKAGES_i386_mingw/9618/

Reply:
Yes. The logic is a bit gnarly. Working on it.